### PR TITLE
add tenant_profile to publicProfile

### PR DESCRIPTION
**Pull request by lxngxr**
* add tenant_profile to publicProfile 
* renamed facebook_link to facebook_url for consistency (will update front-gateway too)
* fallback to FB login only if tenant profile not set

**only merge when front gateway PR is approved**

connects to https://github.com/dorbel-tech/dorbel-app/issues/782

### CODE REVIEW

File: src/utils/userManagement.js [hunk 1]
```diff
@@ -95,11 +95,11 @@ function getPublicProfile(user_uuid) {
       last_name: _.get(user, 'user_metadata.last_name') || user.family_name,
       phone: _.get(user, 'user_metadata.phone'),
       picture: user.picture,
-      role: _.get(user, 'app_metadata.role')
+      tenant_profile: _.get(user, 'user_metadata.tenant_profile'),
     };
 
-    if (_.get(user, 'identities[0].provider') === 'facebook') {
-      publicProfile.facebook_link = user.link;
+    if (!user.tenant_profile && _.get(user, 'identities[0].provider') === 'facebook') {
```
Comment (virtser) [inline src/utils/userManagement.js]: user.user_metadata.tenant_profile?

Comment (lxngxr) [inline src/utils/userManagement.js]: :+1: Good catch - I meant publicProfile.tanant_profile...

File: src/utils/userManagement.js [hunk 2]
```diff
@@ -95,11 +95,11 @@ function getPublicProfile(user_uuid) {
...
       phone: _.get(user, 'user_metadata.phone'),
       picture: user.picture,
-      role: _.get(user, 'app_metadata.role')
+      tenant_profile: _.get(user, 'user_metadata.tenant_profile'),
     };
 
-    if (_.get(user, 'identities[0].provider') === 'facebook') {
-      publicProfile.facebook_link = user.link;
+    if (!publicProfile.tenant_profile && _.get(user, 'identities[0].provider') === 'facebook') {
+      publicProfile.facebook_url = user.link;
```
Comment (virtser) [inline src/utils/userManagement.js]: publicProfile.tenant_profile.facebook_url

File: src/utils/userManagement.js [hunk 3]
```diff
@@ -95,11 +95,11 @@ function getPublicProfile(user_uuid) {
       last_name: _.get(user, 'user_metadata.last_name') || user.family_name,
       phone: _.get(user, 'user_metadata.phone'),
       picture: user.picture,
-      role: _.get(user, 'app_metadata.role')
+      tenant_profile: _.get(user, 'user_metadata.tenant_profile'),
     };
 
-    if (_.get(user, 'identities[0].provider') === 'facebook') {
-      publicProfile.facebook_link = user.link;
+    if (!publicProfile.tenant_profile && _.get(user, 'identities[0].provider') === 'facebook') {
```
Comment (virtser) [inline src/utils/userManagement.js]: !publicProfile.tenant_profile.facebook_url

Comment (lxngxr) [inline src/utils/userManagement.js]: This will prevent users from editing/removing the FB link as we discussed

**Pull request merged**